### PR TITLE
fixes: LT-2762, LT-2532, LT-2730, LT-1601

### PR DESCRIPTION
**Pull request by jfaracik**
**What's fixed**
* LT-2762 - Back button doesn't work on generate documents page
* LT-2532 - The icon for open queries shows when there are no open queries only closed
* LT-2730 - Clearance level is missing on F680 applications
* LT-1601 - Good review is not stored

### CODE REVIEW

File: templates/case/slices/destinations.html [hunk 1]
```diff
@@ -47,6 +53,18 @@ <h2 class="govuk-heading-m">{{ destinations|length }} destination{{ destinations
 								<span aria-hidden="false" class="govuk-table__header">Name</span>
 								{{ destination.name }}
 							</td>
+							{% if destinations|list_has_property:'clearance_level' %}
+								<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">
+									<span aria-hidden="false" class="govuk-table__header">Clearance</span>
```
Comment (efahy) [inline templates/case/slices/destinations.html:58]: LCS please

File: templates/case/slices/destinations.html [hunk 2]
```diff
@@ -47,6 +53,18 @@ <h2 class="govuk-heading-m">{{ destinations|length }} destination{{ destinations
...
+									{{ destination.clearance_level.value }}
+								</td>
+							{% endif %}
+							{% if destinations|list_has_property:'descriptors' %}
+								<td class="govuk-table__cell {% if show_advice %}lite-!-no-border{% endif %}">
+									<span aria-hidden="false" class="govuk-table__header">Descriptors</span>
```
Comment (efahy) [inline templates/case/slices/destinations.html:64]: lite-content

File: templates/components/picklist-picker.html [hunk 3]
```diff
@@ -26,19 +26,17 @@
...
-			Change {{ component.type|sentence_case|lower }}
-		</a>
-		<input type="hidden" id="{{ component.target }}-input" name="{{ component.target }}" value="{{ value|get:component.target }}">
-		<div id="{{ component.target }}-text" class="govuk-inset-text govuk-!-margin-top-0 {% if not value|get:component_text %}app-hidden{% endif %}">
-			{{ value|get:component_text }}
-		</div>
-	{% endwith %}
+	<a id="link-{{ component.target }}-picklist-picker" data-picklist-target="{{ component.target }}" data-picklist-set-text="false" href="#" class="govuk-link govuk-link--no-visited-state app-picklist-picker-heading__link {% if value|get:component.target %}app-hidden{% endif %}">
+		{% svg 'plus' %}
+		Use {{ component.type|sentence_case|lower }} from picklist
```
Comment (efahy) [inline templates/components/picklist-picker.html:31]: lite-content?

File: templates/case/slices/destinations.html (hunk 1 above)
Comment (jfaracik) [reply templates/case/slices/destinations.html]: Done!

**Pull request merged**